### DISCUSSION TITLE
Automatic localization

### DISCUSSION THREAD

**Discussion by soulsands** [TriliumNext/Trilium#3830] [General]
Hi, an idea comes to me that  we can use chatgpt for localization. It's quite easy to interact with chatgpt by their api. It's not expensive and provides quite high quality transilation.I can offer my free token if needed.
Then what we need to do is just to collect target texts. we can wrap all target texts with a flag function call, use RegExp to extract them all and finally build a json map file. Other concerns might be caching to update the map incremantally or splicing the map for lazy loading.

**Comment by zadam** [comment]
Hello, that's a neat idea, but at this point I don't plan to add support for localization, see: https://github.com/zadam/trilium/wiki/FAQ#translation--localization-support
